.github/workflows/preview-create.yml

```diff
@@ -25,14 +25,17 @@ on:
         description: Domain which these preview environments reside on.
       post_deploy_command:
         type: string
-        description: Command which will be executed after the environment is created. 
+        description: Command which will be executed after the environment is created.
     secrets:
       skpr_preview_registry_url:
         required: true
       skpr_preview_username:
         required: true
       skpr_preview_password:
         required: true
+      node_auth_token:
+        required: false
```

Comment:
Required: false seems to be ignored in some cases. Probably needs testing that it doesn't break some builds.